lutris/gui/application.py

```diff
@@ -583,11 +583,18 @@ def print_game_json(self, command_line, game_list):
                 "slug": game["slug"],
                 "name": game["name"],
                 "runner": game["runner"],
-                "platform": game["platform"],
-                "year": game["year"],
-                "playtime": str(timedelta(hours=game["playtime"])),
-                "lastplayed": str(datetime.fromtimestamp(game["lastplayed"])),
-                "directory": game["directory"],
+                "platform": game["platform"] or None,
+                "year": game["year"] or None,
+                "directory": game["directory"] or None,
+                "hidden": True if game["hidden"] else False,
```

Comment:
Isn't game["hidden"] already a boolean value? Why the if?

Reply:
The value of `hidden` from the database is either a `1` or a `0` and not a boolean. Line 589 just replaces `1` with `True` and `0` with `False`.

In the [original issue](https://github.com/lutris/lutris/issues/3379) where I asked for more fields in the JSON output, @strycore [asked for the JSON output to be human readable](https://github.com/lutris/lutris/issues/3379#issuecomment-769974455). I think a boolean is a lot more readable than 1s or 0s.

Reply:
I could change Line 589  to be `"hidden": bool(game["hidden"])`, but I think using a ternary makes it easier to understand what the line does.